Mathlib/MeasureTheory/Measure/MeasureSpace.lean

```diff
@@ -1074,7 +1074,8 @@ theorem mapₗ_congr {f g : α → β} (hf : Measurable f) (hg : Measurable g) (
 open Classical in
 /-- The pushforward of a measure. It is defined to be `0` if `f` is not an almost everywhere
 measurable function. -/
-irreducible_def map [MeasurableSpace α] (f : α → β) (μ : Measure α) : Measure β :=
+irreducible_def map [MeasurableSpace α] [MeasurableSpace β] (f : α → β) (μ : Measure α) :
```

Comment:
Should `MeasurableSpace α` be found by unification here?

Reply:
> Should `MeasurableSpace α` be found by unification here?

It can't be found by unfication if `μ` isn't provided and `f` is a polymorphic function. It happens quite often in mathlib, as I discovered in a previous PR similar to this one but for the `MeasurableSpace α` arguments.

> Should we add a notation for "`map` with a given σ-algebra in the codomain"?

I would like to, but I don't know what it should be. `map[mα, mβ]` sounds... ambiguous. It would be great if we could somehow generally make `f[a, b]` notation for `@f _ _ a _ b` where the `a` and `b` go fill in the first instance arguments of `f`.

For now, I suggest we merge this PR without any new notation